src/handlers/event-strategies/default-event-strategy.ts

```diff
@@ -14,6 +15,24 @@ export class DefaultEventStrategy implements IEventStrategy<Event, Promise<void>
     private readonly eventRepository: IEventRepository,
   ) { }
 
+  protected async isEventExpired(event: Event): Promise<Event | ExpiredEvent> {
+    const [, rawExpirationTime] = event.tags.find((tag) => tag.length === 2 && tag[0] === EventTags.Expiration)
```

Comment:
should probably check that the tag length is >= 2, the rest of the elements are safe to ignore but the event would still be valid

Reply:
would add `Number.isSafeInteger(Number(tag[1]))`